### PR TITLE
Add link to github.com/username on picture

### PR DESCRIPTION
**Pull request by Hoenn**
# Fixes :shipit: :shipit: :shipit:
Nothing in particular, just an enhancement. Searched a friend and tried to click their picture to get to their page, figured it was in the spirit of the event to make that happen.

(possibly https://github.com/jenkoian/hacktoberfest-checker/issues/128)

# Summary :1st_place_medal: :1st_place_medal: :1st_place_medal:
Adds a link to the searched user's github page to the their profile picture that appears after searching.

### CODE REVIEW

**Comment by jenkoian** [comment]
Brilliant PR @Hoenn :) Mind fixing the conflicts?

File: views/partials/prs.hbs [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {{#exists prs}}
     <div class="tc white">
         {{#if userImage}}
-            <img class="br-100 w4" height="128" src="{{userImage}}" width="128">
+            <a href = "https://github.com/{{username}}">
```
Comment (jenkoian) [inline views/partials/prs.hbs]: I think we should open this in a new tab/window. You ok to add `target="_blank"`?

Comment (Hoenn) [reply views/partials/prs.hbs]: I'm not super fond of that behavior, I usually middle-click a link for a new tab. If you think it's better then I'll change that!

Comment (jenkoian) [reply views/partials/prs.hbs]: Yeah me neither, just thinking on mobile, if people actually press the face...what do you think?

Comment (Hoenn) [reply views/partials/prs.hbs]: Good point! New tab seems like a fair middleground

**Comment by jenkoian** [comment]
Thanks @Hoenn !

**Pull request merged**